packages/components/src/models/UnoCard.ts

```diff
@@ -0,0 +1,32 @@
+import type { Card } from '@models/Card';
+
+export const ALL_UNO_COLORS = ['red', 'blue', 'green', 'yellow'] as const;
+export type UnoCardColor = typeof ALL_UNO_COLORS[number];
+
+export const ALL_UNO_VALUES = [
+  '0',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9',
+  'plus2',
+  'plus4',
+  'jump',
+  'reverse',
+  'joker',
+] as const;
+export type UnoCardValue = typeof ALL_UNO_VALUES[number];
```

Comment:
minor: youd didn't change those variables to pascal case in the end?

Reply:
I forgot that one :)